ksql-rest-app/src/main/java/io/confluent/ksql/rest/server/computation/Command.java

```diff
@@ -25,15 +25,19 @@
 @JsonSubTypes({})
 public class Command {
   private final String statement;
+  private final boolean isNewQueryIdGeneration;
   private final Map<String, Object> overwriteProperties;
   private final Map<String, String> originalProperties;
   private final boolean preVersion5;
 
   @JsonCreator
   public Command(@JsonProperty("statement") final String statement,
+                 @JsonProperty("isNewQueryIdGeneration") final Boolean isNewQueryIdGeneration,
                  @JsonProperty("streamsProperties") final Map<String, Object> overwriteProperties,
                  @JsonProperty("originalProperties") final Map<String, String> originalProperties) {
     this.statement = statement;
+    this.isNewQueryIdGeneration =
```

Comment:
you could just do `this.isNewQueryIdGeneration = (isNewQueryIdGeneration != null)` but I suppose that's a little tricky icky icky 😄